ApplicationLibCode/Commands/RicCreateEnsembleWellLogFeature.cpp

```diff
@@ -217,12 +217,11 @@ void RicCreateEnsembleWellLogFeature::executeCommand( const RicCreateEnsembleWel
                 allLasFileNames << lasFile;
         }
 
-        // Remove the temporary plots after export
-        plotCollection->removePlot( wellLogPlot );
-
         RicCloseCaseFeature::deleteEclipseCase( eclipseCase );
     }
 
+    plotCollection->deleteAllPlots();
```

Comment:
This will delete the other (unrelated) plots if the user is running the import on an existing project. Maybe to a real life problem?